services/movie_session.py

```diff
@@ -42,3 +42,18 @@ def update_movie_session(
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
```

Comment:
Consider handling the `DoesNotExist` exception for the `MovieSession.objects.get(id=session_id).delete()` call to manage cases where a session with the given ID does not exist. This will prevent the application from crashing due to an unhandled exception.